lib/slope_intercept_conversion.rb

```diff
@@ -0,0 +1,22 @@
+require 'cosmos/conversions/conversion'
+
+module Cosmos
+  class SlopeInterceptConversion < Conversion
+    def initialize(packet_field, slope, intercept=0)
+      super()
+      @converted_type = :FLOAT
+      @converted_bit_size = 32
+      @packet_field = packet_field
+      @slope = slope.to_f
+      @intercept = intercept.to_f
+    end
+    def call(value, packet, buffer)
+      v =  * 2.50/1024
```

Comment:
Where is v used later?  If it is not, delete this line.

Reply:
Good catch! Resolved in commit 1aeae737cb14bc2b690a2c5f6c9a487e6fc18bc8